tests/pruners_tests/test_percentile.py

```diff
@@ -228,5 +228,7 @@ def setup_study(trial_num: int, _intermediate_values: List[List[float]]) -> Stud
     all_trials = study.get_trials()
     direction = study.direction
     assert math.isnan(
-        _percentile._get_percentile_intermediate_result_over_trials(all_trials, direction, 2, 75)
+        _percentile._get_percentile_intermediate_result_over_trials(
+            all_trials, direction, 2, 75, 1
+        )
     )
```

Comment:
Could you add a test case for `n_min_trials=2`?

Reply:
Sure! https://github.com/optuna/optuna/pull/2556/commits/ecd54f39daf58c6f53965b0bfd5610b47ced6436